addons/mass_mailing_sale/models/mailing_mailing.py

```diff
@@ -16,13 +16,13 @@ class MassMailing(models.Model):
     def _compute_sale_quotation_count(self):
         has_so_access = self.env['sale.order'].check_access_rights('read', raise_exception=False)
         for mass_mailing in self:
-            mass_mailing.sale_quotation_count = self.env['sale.order'].search_count(self._get_sale_utm_domain()) if has_so_access else 0
+            mass_mailing.sale_quotation_count = self.env['sale.order'].search_count(mass_mailing._get_sale_utm_domain()) if has_so_access else 0
```

Comment:
Seems like a fix for stable, isn't it ? At least to be done in a separate commit